setup.py

```diff
@@ -1,49 +1,39 @@
-from setuptools import setup, find_packages
+from setuptools import find_packages
+from setuptools import setup
 
-setup(name='collective.honeypot',
-      version='1.0.4.dev0',
-      description="Anti-spam honeypot for Plone",
-      long_description=(open("README.rst").read() + "\n" +
-                        open("CHANGES.rst").read()),
-      # Get more strings from https://pypi.org/classifiers
-      classifiers=[
-          "Development Status :: 5 - Production/Stable",
-          "Framework :: Plone",
-          "Framework :: Plone :: 3.3",
-          "Framework :: Plone :: 4.0",
-          "Framework :: Plone :: 4.1",
-          "Framework :: Plone :: 4.2",
-          "Framework :: Plone :: 4.3",
-          "License :: OSI Approved :: GNU General Public License (GPL)",
-          "Programming Language :: Python",
-          "Programming Language :: Python :: 2.4",
-          "Programming Language :: Python :: 2.6",
-          "Programming Language :: Python :: 2.7",
-          ],
-      keywords='honeypot antispam form protection plone',
-      author='Maurits van Rees',
-      author_email='maurits@vanrees.org',
-      url='https://github.com/collective/collective.honeypot',
-      license='GPL',
-      packages=find_packages(exclude=['ez_setup']),
-      namespace_packages=['collective'],
-      include_package_data=True,
-      zip_safe=False,
-      install_requires=[
-          'setuptools',
-      ],
-      extras_require = {
-          'test': [
-              'plone.app.testing',
-              ],
-          'fixes': [
-              'z3c.jbot',
-              ],
-      },
-      entry_points="""
+
+setup(
+    name="collective.honeypot",
+    version="2.0.ev0",
+    description="Anti-spam honeypot for Plone",
+    long_description=(open("README.rst").read() + "\n" + open("CHANGES.rst").read()),
+    # Get more strings from https://pypi.org/classifiers
+    classifiers=[
+        "Development Status :: 5 - Production/Stable",
+        "Framework :: Plone",
+        "Framework :: Plone :: 5.2",
+        "License :: OSI Approved :: GNU General Public License (GPL)",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+    ],
+    keywords="honeypot antispam form protection plone",
+    author="Maurits van Rees",
+    author_email="maurits@vanrees.org",
+    url="https://github.com/collective/collective.honeypot",
+    license="GPL",
+    packages=find_packages(),
+    namespace_packages=["collective"],
+    include_package_data=True,
+    zip_safe=False,
+    install_requires=["setuptools",],
+    extras_require={"test": ["plone.app.testing",], "fixes": ["z3c.jbot",],},
```

Comment:
Probably the `fixes` extra can go away, and `z3c.jbot` moved to `install_requires`.